mdk/README.txt

```diff
@@ -56,7 +56,7 @@ installation process, no further action is required on your part.
 
 LexManos' Install Video
 =======================
-https://www.youtube.com/watch?v=8VEdtQLuLO0&feature=youtu.be
+https://youtu.be/8VEdtQLuLO0
 
 For more details update more often refer to the Forge Forums:
-http://www.minecraftforge.net/forum/index.php/topic,14048.0.html
+http://www.minecraftforge.net/forum/topic/13860-tutorial-getting-started-with-forgegradle/
```

Comment:
The contents of this link (and the grammar of the heading) are pretty outdated/broken. Myabe we just don't need this one at all?

Reply:
I think that a link about using Forge gradle is necessary, a link to some documentation would be good rather than the broken forum post though

Reply:
There's a link to the forge docs at the very top.